### DISCUSSION TITLE
이벤트 핸들러의 콜백을 어떻게 처리할것인가?

### DISCUSSION THREAD

**Discussion by hwangstar156** [woowacourse-teams/2022-gong-seek#642] [refactor]
현재 onClick내에 1줄일 경우 인라인으로 작성하기로 했습니다. 하지만 혼용되고 있는 부분이 있습니다. 한번 더 컨벤션을 맞추는게 좋을것 같습니다!

**Comment by hwangstar156** [comment]
개인적으로는 이벤트 핸들러의 콜백은 모두 handle... 이라는 네이밍으로 빼는게 좋을것 같습니다.

현재 inline으로 넣어서 처리하니 전체적인 jsx의 가독성이 떨어지는것같습니다. 또한 1줄이상일때는 빼서 관리하고 1줄일때는 인라인으로 작성하는데 이것이 컴포넌트내에서 섞이게 되어서 혼잡해보입니다.

**Comment by liswktjs** [comment]
기존에 인라인 함수로 들어가 있던 것들을 
[명사][이벤트이름][동사] 
handle[이벤트]Click 
ex) handleLoginButtonClick vs onLoginButtonClick

**Comment by hwangstar156** [comment]
https://blog.sonim1.com/220

**Comment by liswktjs** [comment]
onClick props와의 대비를 위해서 handle(돔)Click식으로 네이밍 지정
